README.md

```diff
@@ -20,6 +20,7 @@ Here are some examples that walk you through how to use the client: [doc/example
 
 | Typesense Server | typesense-php |
 |------------------|----------------|
+| \>= v0.16.0 | \>= v4.1.0 |
```

Comment:
since some of the changes will break earlier implementations of the client, I guess we need to create a major update for the client with each major update of the server.

Reply:
I've added default values for the new `options` parameter in `import` and `createMany` will still work in this version, by calling `import` internally. So this version should not break any earlier implementations. Let me know if you notice any other changes that might be backward incompatible.